### PR TITLE
Added a new "endOfLifeDate" property for dotnet runtime entries.

### PR DESCRIPTION
**Pull request by kshyju**
Fixes https://github.com/Azure/azure-functions-dotnet-worker/issues/1075

Added a new `endOfLifeDate` property under the `displayInfo` part of dotnet run time entries. We are exposing this as an optional property and will share that information with the consumers of this feed. No value for .NET Framework entry as we do not specify the version of .NET framework in the dropdown VS shows when creating a new function app.

This change is added in a patch version. So I cloned **4.26.0**, renamed to 4.26.1 and made the changes to those entries. Udpated the `tags=>v4-prerelease` element value to use this new patch version.

The date value used is ISO 8601 format version of dates (Printed using code like below in c#). 

````
var d = DateTime.SpecifyKind(new DateTime(2022, 10, 11), DateTimeKind.Utc);

// ToString in ISO 8601 format.
var str = d.ToString("O");
str.Dump();

// Enusre this is parsable
DateTime.Parse(str).Dump();
````

### CODE REVIEW

File: cli-feed-v4.json
Comment (kshyju) [inline cli-feed-v4.json:9644]: @cloudmelon Could you double check the dates?

**Review by soninaren — APPROVED** [review]

File: cli-feed-v4.json
Comment (cloudmelon) [reply cli-feed-v4.json:9644]: it is when Windows is supported. I think we have windows 10 and windows 11

**Comment by phenning** [comment]
Is a goal here to have tooling which consumes the feed to automatically mark as out of support (or perhaps hide) these variants which have an endOfLifeDate earlier then the current date?

**Comment by cloudmelon** [comment]
> goal here to have tooling which consumes the feed to automatically mark as out of support (or perhaps hide) these variants which have an endOfLifeDate ea

Yes. @phenning That's what I am trying to help the team achieve, and have some good leads. there's an existing release.json for .NET / .NET Core, but the .NET framework is a bit tricky as it's tied up to the lifecycle of the installed Windows OS edition.

**Comment by kshyju** [comment]
Blocked until @michaelpeng36 PR is merged. Will rebase this PR after that. 

Michael's PR is blocked by AppVeyor pipeline issues.  @kashimiz Is this one of the repos you are migrating away from appveyor?

**Comment by kshyju** [comment]
@soninaren Rebased. Can you review again please? As suggested by @ejizba, I updated only prerelease tag in this PR. Will do a follow PR for updating the GA tag entry after a few days.

**Review by soninaren — APPROVED** [review]

**Pull request merged**
